iap/refresh/js/poll.js

```diff
@@ -0,0 +1,68 @@
+// Copyright Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+(function poll() {
+  setTimeout(function() {
```

Comment:
I'd prefer you give this function a name and invoke setTimeout separately, e.g.:

```
function pollForLoginState() {
    ...
}

setTimeout(pollForLoginState, 10000);
```

Reply:
Done, changed to setInterval